.github/workflows/end2end.yaml

```diff
@@ -403,6 +408,7 @@ jobs:
         working-directory: ./.github/scripts/end2end
       - name: Debug wait
         uses: ./.github/actions/debug-wait
+        timeout-minutes: 30
```

Comment:
should we keep a timeout here?
this timeout will prevent debugging for more than 30 minutes....

Reply:
extended to 60 here : https://github.com/scality/Zenko/pull/2031/commits/56ab10b06a5d2e2888726530459674314abc8a3d#diff-cf60998e2851400a5ff7faeaf9677e20a46bab4b90a3b4934293e480ca335d2fR411